mitmproxy/tools/web/webaddons.py

```diff
@@ -14,8 +15,8 @@ def load(self, loader):
         loader.add_option(
             "web_columns",
             Sequence[str],
-            ["tls", "icon", "path", "method", "status", "size", "time"],
-            "Columns to show in the flow list",
+            AVAILABLE_WEB_COLUMNS,
+            "all available columns to show in the flow list",
```

Comment:
```suggestion
            f"Columns to show in the flow list. Can be one of the following: {', '.join(AVAILABLE_WEB_COLUMNS)}",
```

(or something along those lines)